surveys/2024-annual-survey/translations/fr.md

```diff
@@ -276,140 +276,140 @@ Rows:
+- Rencontre avec des messages d’erreur de compilateur opaques/peu clairs
+- Prise en charge IDE inférieure à la moyenne (par exemple, certaines erreurs ne sont pas affichées ou l’analyse est lente)
+- Expérience de débogage inférieure à la moyenne (par exemple, visualisations de valeurs manquantes ou traces de pile asynchrones)
+- Manque de documentation sur le langage Rust ou la bibliothèque standard
 
 Columns:
 
-- Haute priorité
-- Priorité moyenne
-- Priorité basse
+- Gros problème pour moi
+- Pourrait être amélioré, mais ne me limite pas
+- Ce n’est pas du tout un problème pour moi
 
 ### Y a-t-il des fonctionnalités non mentionnées ci-dessus que vous aimeriez voir priorisées ?
 
 Type: free form
 
-### 
+### Quelles fonctionnalités stabilisées au cours des 12 derniers mois utilisez-vous le plus ?
 
 Type: matrix
 
 Rows:
 
-- 
-- 
-- 
-- 
-- 
-- 
-- 
-- 
-- 
-- 
-- 
-- 
-- 
-- 
+- [fonction asynchrone dans les traits](https://blog.rust-lang.org/2023/12/21/async-fn-rpit-in-traits.html) (1.75)
+- [Position de retour impl Trait dans traits](https://blog.rust-lang.org/2023/12/21/async-fn-rpit-in-traits.html) (1.75)
+- [Littéraux de chaîne C](https://blog.rust-lang.org/2024/03/21/Rust-1.77.0.html#c-string-literals) (1.77)
+- [Récursivité dans la fonction asynchrone](https://blog.rust-lang.org/2024/03/21/Rust-1.77.0.html#support-for-recursion-in-async-fn) (1.77)
+- [macro offset_of!](https://blog.rust-lang.org/2024/03/21/Rust-1.77.0.html#offset_of) (1.77)
+- [Attribut #[diagnostic]](https://blog.rust-lang.org/2024/05/02/Rust-1.78.0.html#diagnostic-attributes) (1.78)
+- [Expressions constantes en ligne](https://blog.rust-lang.org/2024/06/13/Rust-1.79.0.html#inline-const-expressions) (1.79)
+- [Limites de la position du type associé](https://blog.rust-lang.org/2024/06/13/Rust-1.79.0.html#bounds-in-associated-type-position) (1.79)
+- [LazyCell et LazyLock](https://blog.rust-lang.org/2024/07/25/Rust-1.80.0.html#lazycell-and-lazylock) (1.80)
+- [Gammes exclusives en motifs](https://blog.rust-lang.org/2024/07/25/Rust-1.80.0.html#exclusive-ranges-in-patterns) (1.80)
+- [Trait Error dans core](https://blog.rust-lang.org/2024/09/05/Rust-1.81.0.html#coreerrorerror) (1.81)
+- [#[expect(lint)] et raisons de lint](https://blog.rust-lang.org/2024/09/05/Rust-1.81.0.html#expectlint) (1.81)
+- [Syntaxe de capture précise](https://blog.rust-lang.org/2024/10/17/Rust-1.82.0.html#precise-capturing-use-syntax) (1.82)
+- [pointeurs &raw](https://blog.rust-lang.org/2024/10/17/Rust-1.82.0.html#native-syntax-for-creating-a-raw-pointer) (1.82)
 
 Columns:
 
-- 
-- 
-- 
-- 
-- 
+- J&#39;utilise cette fonctionnalité
+- Je ne peux pas encore utiliser cette fonctionnalité
+- Je n&#39;ai pas besoin de cette fonctionnalité
+- Je ne savais pas qu’elle était stabilisée
+- Je ne sais pas ce que c&#39;est
 
-### Comment compilez-vous vos projets Rust?
+### Comment compilez-vous vos projets Rust ?
```

Comment:
redundant space?

Reply:
No, it's a punctuation mark[^1], we have those before every `?`, `!`, `:` ...

[^1]: https://www.babbel.com/en/magazine/french-spacing

Reply:
Nope, in French there is always a (thin non-breaking) space before the question mark

Reply:
wow. cool!